src/index.html

```diff
@@ -3,39 +3,46 @@
 layout: default
 ---
 
-<section id="deck">
-  <section>
-    <h1>A modern and consistent transportation experience throughout California</h1>
-    <p>
-      Learn how the California Integrated Travel Project (Cal-ITP) is making riding by bus and train simpler and more
-      cost-effective—for providers and customers.
-    </p>
-  </section>
+<section class="row">
+  <div class="col-12 col-lg-5 d-flex align-items-center">
+    <div class="ms-lg-4">
+      <h1 class="text-center text-lg-start pt-5 pt-lg-0">A modern and consistent transportation experience throughout California</h1>
+      <p class="text-center text-lg-start">
+        Learn how the California Integrated Travel Project (Cal-ITP) is making riding by bus and train simpler and more
+        cost-effective—for providers and customers.
+      </p>
+    </div>
+  </div>
 
-  <picture>
-    <img
-      id="triforce"
-      src="images/hero-header.png"
-      alt="A trio of images, clockwise from top: a bus, a train platform with a sign that announces “Next train in 3 minutes,” and a transit rider paying their fare by tapping their smartphone’s mobile wallet on a payment reader when boarding" />
-  </picture>
+  <div class="col-12 offset-lg-1 col-lg-5 d-flex align-items-center justify-content-center">
+    <picture>
+      <img
+        style="max-width: 441px;"

```

Comment:
This feels a little strange, I think the only place we have an inline `style` attribute? Qualms about moving it back into the CSS?

Reply:
No qualms really!